troposphere/templates/allocations.html

```diff
@@ -14,14 +14,11 @@
         </div>
         <div class="message-wrapper">
 
-            <h1>Your account does not currently have a valid Allocation Source.</h1>
+            <h1>You do not currently have an <strong><em>ACTIVE</em></strong> Jetstream allocation.</h1>
```

Comment:
In the spirit of having a kinder message, what if we ask the user to do something rather than tell them that something is wrong? As a new user this message feels like I have already messed up. 

Maybe something like: 
> In order to use this application please create an XSEDE account by visiting xsede.org
> 
> Allocations are required to run instances and are awarded exclusively through the eXtreme Science and Engineering Discovery Environment (XSEDE). You can find more information as well as instructions about how to apply for an XSEDE portal account and an XSED allocation here.